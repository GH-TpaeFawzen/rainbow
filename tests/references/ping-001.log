PING www.google.com(dh-in-x67.1e100.net (2a00:1450:400b:c03::67)) 56 data bytes
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=1 ttl=54 time=36.6 ms
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=2 ttl=54 time=36.6 ms
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=3 ttl=54 time=36.6 ms

--- www.google.com ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2003ms
rtt min/avg/max/mdev = 36.625/36.640/36.666/0.221 ms
