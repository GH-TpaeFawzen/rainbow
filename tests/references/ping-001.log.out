[BOLD]PING www.google.com(dh-in-x67.1e100.net (2a00:1450:400b:c03::67)) 56 data bytes[RESET_INTENSITY]
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=1 ttl=54 [FOREGROUND_YELLOW]time=[BOLD]36.6 ms[RESET_INTENSITY][FOREGROUND_RESET]
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=2 ttl=54 [FOREGROUND_YELLOW]time=[BOLD]36.6 ms[RESET_INTENSITY][FOREGROUND_RESET]
64 bytes from dh-in-x67.1e100.net (2a00:1450:400b:c03::67): icmp_seq=3 ttl=54 [FOREGROUND_YELLOW]time=[BOLD]36.6 ms[RESET_INTENSITY][FOREGROUND_RESET]

[BOLD]--- www.google.com ping statistics ---[RESET_INTENSITY]
3 packets transmitted, 3 received, 0% packet loss, time 2003ms
rtt min/avg/max/mdev = 36.625/36.640/36.666/[BOLD]0.221 ms[RESET_INTENSITY]
